cache/in-memory/src/lib.rs

```diff
@@ -57,33 +57,29 @@ pub mod model;
 
 mod builder;
 mod config;
+mod event;
 mod stats;
-mod updates;
+mod test;
```

Comment:
shouldn't this only compile under test conditions?

Reply:
Thanks, I'll find a way to do that.